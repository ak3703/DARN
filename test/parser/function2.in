bool main(int age) { 
	bool isTrue;
	isTrue = false;
	print(1);
}
