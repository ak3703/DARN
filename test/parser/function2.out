
fdecl { 
bool main(age) {
vdecl { 
bool id isTrue;
}
Assign { isTrue =  bool_lit false };
print(int_lit 1);
} 
}
